pkg/network/setup/netstat.go

```diff
@@ -138,6 +138,15 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		}
 	}
 
+	interfaceByName := netvmispec.IndexInterfaceSpecByName(vmi.Spec.Domain.Devices.Interfaces)
```

Comment:
It was done already and the result is in `vmiInterfacesSpecByName`.

Reply:
vmiInterfacesSpecByName is now reused